client/schemes/jellyfinScheme.ts

```diff
@@ -59,20 +73,21 @@ export default class JellyfinScheme {
     rememberMe: boolean;
   }): Promise<AxiosResponse<AuthenticationResult> | void> {
     // Ditch any leftover local tokens before attempting to log in
-    await this.$auth.reset();
+    await this.$auth.logout();
```

Comment:
`reset()` is how it should be done: https://github.com/nuxt-community/auth-module/blob/master/lib/schemes/local.js#L38

Reply:
Calling logout does the same thing + sends a call to server to revoke the old session, which is something reset doesn't do and I think it's nice to have.

Reply:
The point of `login` is not to log you out of the server, though. It's only meant to, well, log you in.

It'll never be called when there is a user logged in anyway.